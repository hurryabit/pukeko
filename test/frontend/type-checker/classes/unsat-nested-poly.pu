class C a where
  c : a -> a
f : a -> a
f = let g = c in g
