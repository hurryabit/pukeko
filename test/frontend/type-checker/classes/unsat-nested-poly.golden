test/frontend/type-checker/classes/unsat-nested-poly.pu:4:1: cannot solve constraint C a in context ()
